config/config.schema.json

```diff
@@ -1397,6 +1397,10 @@
           "type": "string",
           "description": "The client ID of the first party application that will be used by the refresher"
         },
+        "earlyRefreshFrequency": {
+          "type": "string",
+          "description": "Early refresh interval before credential is eligible for refresh. For testing. 0 means no early refresh. Example: 24h"
```

Comment:
```suggestion
          "description": "Early refresh interval before credential is eligible for refresh. For testing, 0 means no early refresh. Example: 24h"
```

Reply:
Refresh interval is how frequently the refresher loop runs. By default it's 10 minutes